### DISCUSSION TITLE
Welcome to MBPMid2010_GPUFix Discussions!

### DISCUSSION THREAD

**Discussion by julian-poidevin** [julian-poidevin/MBPMid2010_GPUFix#138] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Infinitymaple** [comment]
Do you think this would work on mac os catalina?

**Reply by julian-poidevin** [reply]
Sorry this is not compatible with this OS

**Comment by gregoiremotot** [comment]
Thanks! Merci beaucoup !!!

**Comment by JeanDeTrez** [comment]
Hi
Could you tell me if this application run well with a macbook pro 2011 intel, (OSX El Capitan 10.11.16) ?

**Reply by julian-poidevin** [reply]
Hello, sorry the app is only compatible with MBP Mid 2010 15 inches. The fix does not work for Macbook Pro 2011 😞 .

**Comment by Hillybilly11** [comment]
Hi Julian, I've bought a secondhand MBP 2010 15" and tried nearly everything to avoid getting the black screen. 
Somehow I landed on your page and used the app. I can't believe that this actually works. You helped me out, thank you.

**Reply by julian-poidevin** [reply]
You're welcome @Hillybilly11 !
